src/components/strategies/create/CreateStrategyTypeMenu.tsx

```diff
@@ -30,7 +30,7 @@ const BlockIconTextDesc = ({
       </div>
       <div className={'flex-shrink space-y-6'}>
         <div className={'text-14 font-weight-500'}>{title}</div>
-        <div className={'h-32 text-12 text-white/60'}>{description}</div>
```

Comment:
why remove the hight class? Now the UI is broken - the icon and buttons jump around when switching between "Recurring" and "Disposable"